samples/Chat/src/app/ChatHeader.tsx

```diff
@@ -31,11 +31,14 @@ export const ChatHeader = (props: ChatHeaderProps): JSX.Element => {
   return (
     <Stack horizontal={true} verticalAlign={'center'} horizontalAlign="end" className={chatHeaderContainerStyle}>
       <div className={paneButtonContainerStyle}>
-        <IconButton
-          onRenderIcon={() => (props.isParticipantsDisplayed ? <People20Filled /> : <People20Regular />)}
-          className={mergeStyles({ color: theme.palette.neutralPrimaryAlt })}
-          onClick={() => props.setHideParticipants(props.isParticipantsDisplayed)}
-        />
+        {
+          /* @conditional-compile-remove-from(stable) meeting composite */
```

Comment:
This isn't really meeting composite related.

We didn't add explicit tag to this one (we should):
![image](https://user-images.githubusercontent.com/82062616/154587284-9ff85aee-dc60-4d18-8f84-2df3ff101de3.png)

Reply:
What is this related to?
Would you happen to know?

Reply:
I am pretty sure this is CallWithChat composite related. @JamesBurnside ?

Reply:
This is a chat-composite feature (so this and the participant pane field in the chat composite should be something like `/* @conditional-compile-remove-from(stable) chat-composite-participant-pane*/`)

Reply:
(more info: https://azure.github.io/communication-ui-library/?path=/docs/composites-chat-basicexample--basic-example#participant-pane)
We made this a beta feature because it doesn't work on mobile form factors and didn't want to GA a feature that didn't have full support

Reply:
Done